fuzzers/libafl-fuzz/Makefile.toml

```diff
@@ -0,0 +1,47 @@
+[env]
+PROJECT_DIR = { script = ["pwd"] }
+CARGO_TARGET_DIR = { value = "${PROJECT_DIR}/target", condition = { env_not_set = ["CARGO_TARGET_DIR"] } }
+FUZZER_NAME="fuzzer"
+PROFILE = { value = "release", condition = {env_not_set = ["PROFILE"]} }
+PROFILE_DIR = {value = "release", condition = {env_not_set = ["PROFILE_DIR"] }}
+LLVM_CONFIG = {value = "llvm-config-18", condition = {env_not_set = ["LLVM_CONFIG"] }}
+AFL_DIR_NAME= {value = "./AFLplusplus-stable"}
+AFL_CC_PATH= {value = "${AFL_DIR_NAME}/afl-clang-fast"}
+LIBAFL_FUZZ_PATH={value = "./target/release/libafl-fuzz"}
+#LLVM_CONFIG="llvm-config-18"
+
+
+[tasks.fuzzer]
```

Comment:
Why is this the fuzzer task? Shouldn't it be a build-afl-cc task or something like that?